AutonomousDemo/Position.cs

```diff
@@ -16,10 +16,10 @@ public Position(double xCoordinate, double yCoordinate)
             this.XCoordinate = xCoordinate;
             this.YCoordinate = yCoordinate;
         }
-        public Position TravelToPosition(double meters, Velocity velocity)
+        public Position TravelToPosition(Velocity velocity)
```

Comment:
Why "TravelToPosition"? Really it's just travelling for one second. (Also, do you want it to be "travel for one second"? Should there be a time parameter?)